note formatting (#2207)
SumoLogic/sumologic-documentation: docs/observability/aws/deploy-use-aws-observability/deploy-with-aws-cloudformation/index.md

> 1. Sign on to the AWS Management console.

Reviewer: ```suggestion
1. Sign in to the AWS Management console.
```

1. Choose an option to invoke AWS CloudFormation Template:
   * Click [this URL](https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml) to invoke the latest Sumo Logic AWS CloudFormation template.
   * Download the AWS Observability Solution template (S3 Link for cloudformation template): https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.6.0/sumologic_observability.master.template.yaml to invoke the latest Sumo Logic AWS CloudFormation template.<br/>
:::note

> If you would like to download or inspect this or other versions of this template, please visit [Change Log](../changelog.md). 

Reviewer: ```suggestion
Download this or other versions of this template from [Change Log](../changelog.md). 
```

:::
:::note

>  If you want to change the Collector Name and Source Categories of the Sumo Logic sources that are created by default, download CloudFormation template version 2.1.0 or greater and follow [these instructions](#modify-the-collector-name-and-source-categories) before proceeding.

Reviewer: ```suggestion
To change the Collector Name and Source Categories of Sumo Logic sources, you must download CloudFormation template version 2.1.0 or greater and follow the instructions in the (#modify-the-collector-name-and-source-categories) section.
```
